packages/js/navigation/src/index.js

```diff
@@ -18,16 +18,16 @@ import { getAdminLink } from '@woocommerce/settings';
  * Internal dependencies
  */
 import { getHistory } from './history';
-import * as navUtils from './index';
```

Comment:
This was causing an issue with mocking.  Not sure why it was needed for tests, but all tests are passing without it.

Reply:
Yeah that seems rather strange on why this was suppose to happen.